compiler/include/resolution.h

```diff
@@ -58,6 +58,11 @@ void implementForallIntents2(CallExpr* call, CallExpr* origToLeaderCall);
 void implementForallIntents2wrapper(CallExpr* call, CallExpr* origToLeaderCall);
 void stashPristineCopyOfLeaderIter(FnSymbol* origLeader, bool ignore_isResolved);
 
+// reduce intents
+void cleanupRedRefs(Expr*& redRef1, Expr*& redRef2);
```

Comment:
Could you add comments describing what these (now visible to more of the compiler) functions do?